src/java.base/share/classes/java/io/File.java

```diff
@@ -1635,9 +1639,9 @@ public boolean setWritable(boolean writable) {
      * @return  {@code true} if and only if the operation succeeded.  The
      *          operation will fail if the user does not have permission to
      *          change the access permissions of this abstract pathname.  If
-     *          {@code readable} is {@code false} and the underlying
-     *          file system does not implement a read permission, then the
-     *          operation will fail.
+     *          the underlying file system does not implement a read permission,
+     *          then the operation will return the value of the {@code readable}
```

Comment:
There's a switcharoo here. The method description speaks of the platform not supporting file permissions. The return description changes terminology and speaks of the file system not implementating permissions. I think we'll get these consistent but overall the intent is okay.

Reply:
Reconciled these in 8189784ace0a693a042f86505466806377684f80.